sunpy/database/database.py

```diff
@@ -819,6 +819,24 @@ def download_from_vso_query_result(self, query_result, client=None,
         self.add_many(self._download_and_collect_entries(
             query_result, client=client, path=path, progress=progress, overwrite=overwrite))
 
+    def download_from_hek_query_result(self,query_result, path=None,
+                                       progress=False):
+        """Download_from_hek_query_result(query_result,path=None, progress=False)
+        Add new database entries from a HEK query by translating it into a VSO query
+        and invoking the [download_from_vso_query_result] method.
+
+        Parameters
+        ----------
+        query_result : sunpy.net.vso.QueryResponse
+            A VSO query response that was returned by the ``query`` method of a
+            :class:`sunpy.net.vso.VSOClient` object.
+        path , progress
```

Comment:
What are these?

Reply:
Sorry for the late reply, I'll go through, we had an internet outage and slowdown  in Sri Lanka, these are parameters for the method "download_from_vso_query_result".

Reply:
That's fine, there is no rush. 

If that is the case, we need to expand on these items (adding type and a description) like we have for `query_result`.